addons/purchase/migrations/10.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import (division as _py3_division,
```

Comment:
Why these imports?

Reply:
That's jus the standard header I get whenever I create a new .py, I guess the only that matters in this file is the absolute import.  Should I remove it all together or leave some?

Reply:
Nothing is needed here, so remove it completely